requirements.txt

```diff
@@ -15,6 +15,7 @@ texttable
 oauth2client
 google-api-python-client==2.55.0
 google-cloud-storage==2.4.0
+openstacksdk~=1.1.0
```

Comment:
just to double-check `~=` means "upgrade minor versions " , right ?

Reply:
> just to double-check `~=` means "upgrade minor versions " , right ?

Yes.